lib/smart_city/ingestion.ex

```diff
@@ -1,5 +1,6 @@
 defmodule SmartCity.Ingestion do
   alias SmartCity.Helpers
+  alias SmartCity.Ingestion.Transformation
 
   @moduledoc """
   Struct defining an ingestion update event.
```

Comment:
Should we add the transformations to the comments here?

Reply:
+1

Reply:
Done :)